core/src/main/java/org/elasticsearch/index/translog/TranslogWriter.java

```diff
@@ -191,6 +205,14 @@ public int totalOperations() {
         return operationCounter;
     }
 
+    public long getMinSeqNo() {
```

Comment:
I think we can wait and see the usage (but document the caveat - other stats are not consistent either). At the moment it's only used for snapshots which is created under lock. I *think* that the future for recovery/translog trimming will not care about the the writer's values. An alternative is to not expose these but rather the last checkpoint, which is consistent.